README.md

```diff
@@ -1,52 +1,82 @@
 # uniform-codegen
-Code generator utilities including TypeScript types for components for Uniform Projects
+Unform-codegen generates Typescript types for your Uniform project's components. With this, you speed up the local development of your components.
 
 # Installation
 
-Install the `@davetayls/uniform-codegen` package using your preferred package manager.
+Install `@davetayls/uniform-codegen` in your project using your preferred package manager. i.e.
+
+```shell
+npm i -D @davetayls/uniform-codegen
+
+#or
+
+yarn add -D @davetayls/uniform-codegen
+```
 
 # Usage
+To use the codegen, specify the generation command in the package.json file or use a package runner like npx.
+
+In `package.json`, add a new script command that generates types for your components:
 
-The following commands will be available to use via package.json scripts or
-with the `yarn` or `npx` commands.
+```json
+"uniform-codegen": "uniform-codegen components <path-to-components-directory> <full-path-of-output-file>"
```

Comment:
I would make the script name `uniform:codegen` so there is no confusion between the `npm run` option and the `npx` option.

Reply:
I'll update this